.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: generic
+sudo: false
+services:
+- docker
+env:
+  global:
+  - SYSL2_VERSION=0.2.5-ubuntu18.04
```

Comment:
if you want to change the version, you will have to update the travis file.

Reply:
Added image build script